examples/counter-next/shared/src/bin/crux_cli.rs

```diff
@@ -0,0 +1,8 @@
+use std::env;
+
+use uniffi::deps::anyhow;
```

Comment:
Does the core still need to depend on `uniffi` anyway or can it depend on `uniffi` through `crux_core` with the right flags? 🤔 Not really a blocker to merge this, just wondering how far we can take the taking ownership / hiding of our underlying tools of choice.